Examples/Swift/ViewController.swift

```diff
@@ -162,10 +162,10 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         
         exampleMode = .default
         
-        let navigationViewController = NavigationViewController(for: route, locationManager: locationManager())
+        let navigationViewController = NavigationViewController(for: route, styles: [DefaultStyle()], locationManager: locationManager())
```

Comment:
This is sort of a confusing case. It's sort of half the default case since we're not passing in the default dark style? Thoughts on reverting this change?

Reply:
Ah, yeah, this was just to force the light style while working on it.